kms/src/main/java/io/kroxylicious/kms/service/De.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kms.service;
+
+import java.nio.ByteBuffer;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/**
+ * A ByteBuffer-sympathetic deserializer for some type, {@code T}.
+ * @param <T> The type of the deserialized object.
+ * @see Ser
+ */
+public interface De<T> {
```

Comment:
I'm not a fan of the abbreviated class name.

Reply:
I've been tempted to merge them into a single `Serde` type which would solve this naming problem, and also make it clearer that they need to be supplied as a pair. Wdyt?

Reply:
I had been considering a merge too, hadn't quite decided if I thought a merged impl and independent interfaces was the right compromise